test/functional/fixtures/quarantine-mode/quarantine-mode-test.js

```diff
@@ -0,0 +1,98 @@
+var expect         = require('chai').expect;
```

Comment:
usually we name this file`index-test.js`. To avoid duplication in `/quarantine-mode/quarantine-mode-test.js`